AmericanMedicalAssociation/ama-style-guide-2
# EWL-11129: Update morning rounds promo homepage design.

# Pre-PR Checklist
If you have access to approved developer tools such as AMA co-pilot, ask it the following and test any suggestions before posting the PR.


**Jira Ticket**
- [EWL-11129: AMAOne Design Update: Morning Rounds homepage promo](https://ama-it.atlassian.net/browse/EWL-11129)


## Description
Adjust styling of morning rounds homepage promos.


## To Test
- checkout and link ama d8 pr locally: https://github.com/AmericanMedicalAssociation/ama-d8/pull/5148
- clear caches
- on homepage confirm morning rounds promo now stretches full width of the window and matches the following design: https://www.figma.com/design/6ZIRwcG30h03EqhR2sEVf1/Horizon-1?node-id=41-176&node-type=canvas&t=FRZtALqLmzHUgKok-0
- **NOTE** Fonts are being updated as a part of https://ama-it.atlassian.net/browse/EWL-11140


## Visual Regressions
Ensure any available regression testing has been run.


## Relevant Screenshots/GIFs
Use something like [Skitch](https://evernote.com/skitch/) or [GIPHY Capture](https://giphy.com/apps/giphycapture) to capture images/gifs to demonstrate behaviors.


## Remaining Tasks



## Additional Notes
Anything more to add?

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
